### ISSUE TITLE
Request: No hits through 'walls'

### ISSUE THREAD

**Issue by fred0r** [s1lentq/ReGameDLL_CS#314]
it would be nice to be able to completely enable/disable hits through walls, but still allowing hits through wood-boxes/wood-doors.
atm i'm thinking about dust2 on b ct's hitting through 'left side wall' on all the players which are currently running to b.

**Comment by WPMGPRoSToTeMa** [comment]
That's mostly because TraceLine is bugged, but it can be fixed in the engine. Also bullet penetration code isn't correct, it doesn't really measure "wall" thickness.

**Comment by fred0r** [comment]
So this may be possible ?

**Comment by WPMGPRoSToTeMa** [comment]
Yes, I investigated this and also made proof of concept 2-3 years ago. It also required server-side changes of material types bound to textures used by maps (so wood is really wood, not concrete).

**Comment by fred0r** [comment]
It would be nice if you could implement this, best would be to make it configurable (`0/1`)

**Comment by RauliTop** [comment]
> Yes, I investigated this and also made proof of concept (not yet published anywhere, sorry) 2-3 years ago. It also required server-side changes of material types bound to textures used by maps (so wood is really wood, not concrete).

Would be great to have published it 🙄

**Comment by fred0r** [comment]
Sorry but:
Are there any news on this ?
